documentation: add more info on readme

vcapretz opened the pull request:
add info about how to run the project in three different ways
add introduction about the config file
add CoC and Contributing files to allow people contributing to this
project

GitCop commented:
There are some issues with your commit and our naming convention

 * Commit: 0001c001ec3954f3dee780479dcca7bf370c8b4b
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

augusto-jm-amaral commented:
@vcapretz  I added some tables with descriptions of the parameters, then check if the text is correct please.

vcapretz commented:
nice @augusto-jm-amaral, I just changed small stuff and the order of each section. we have to remember that somethings will change after #9 (the example file for instance) 

@MarcoWorms we changed the section order, feel free to approve again 😄
